gc3libs/etc/gcsvBulk_wrapper.m

```diff
@@ -0,0 +1,74 @@
+function gcsvBulk_wrapper(function_name, csvFile, limit, offset)
+%MATLAB_CSV_WRAPPER executes a given function with a given csv file with each line as
+% parameters for the function. The parameters of the csv are limited by limit and are
+% offseted by offset
```

Comment:
"limited by limit and offseted by offset" really sounds like a tautology...  

What does "limit" limit? the number of lines that are read? or the number of characters? or ...?

Likewise, is "offset" a number of lines?